src/main.ts

```diff
@@ -86,6 +87,9 @@ function loadTscConfig(args: string[]):
     return {errors};
   }
 
+  // Store file arguments
```

Comment:
This comment is redundant with the code that immediately follows it, don't you think?